cmd/flux/get.go

```diff
@@ -33,3 +44,71 @@ func init() {
 		"list the requested object(s) across all namespaces")
 	rootCmd.AddCommand(getCmd)
 }
+
+type summarisable interface {
+	AsObject() runtime.Object
+	Len() int
+	SummariseAt(i int, includeNamespace bool) []string
+	Headers(includeNamespace bool) []string
+}
+
+// --- these help with implementations of summarisable
+
+func statusAndMessage(conditions []metav1.Condition) (string, string) {
+	if c := apimeta.FindStatusCondition(conditions, meta.ReadyCondition); c != nil {
+		return string(c.Status), c.Message
+	}
+	return string(metav1.ConditionFalse), "waiting to be reconciled"
+}
+
+type named interface {
+	GetName() string
+	GetNamespace() string
+}
+
+func nameColumns(item named, includeNamespace bool) []string {
+	if includeNamespace {
+		return []string{item.GetNamespace(), item.GetName()}
+	}
+	return []string{item.GetName()}
+}
+
+var namespaceHeader = []string{"Namespace"}
+
+type getCommand struct {
+	headers []string
+	list    summarisable
+}
+
+func (get getCommand) run(cmd *cobra.Command, args []string) error {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	kubeClient, err := utils.KubeClient(kubeconfig, kubecontext)
+	if err != nil {
+		return err
+	}
+
+	var listOpts []client.ListOption
+	if !allNamespaces {
+		listOpts = append(listOpts, client.InNamespace(namespace))
+	}
+	err = kubeClient.List(ctx, get.list.AsObject(), listOpts...)
+	if err != nil {
+		return err
+	}
+
+	if get.list.Len() == 0 {
+		logger.Failuref("no imagerepository objects found in %s namespace", namespace)
```

Comment:
```suggestion
		logger.Failuref("no ImageRepository objects found in %s namespace", namespace)
```

(I aligned this awhile ago as we were using a mixture of camelcase and lowercase names throughout the codebase).